### PR TITLE
fix(link) Broken Links, Counter and ButtonHoles

### PR DESCRIPTION
**Pull request by bobgeorgethe3rd**
Noticed this when trying to look it up. The counter page exists it is just not registered as count so the link needed changing :)

Edit: Noticed the buttonhole pages had broken links as well

Changes
- Changed link from "count" to "counter"
- Added api to buttonhole links

### CODE REVIEW

File: markdown/dev/reference/api/config/options/en.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ There are the five option types that an aspiring pattern designer should be
 familiar with:
 
 1. [**boolean** options][bool] are for yes/no choices
-2. [**counter** options][count] are for integer values
+2. [**counter** options][counter] are for integer values
```
Comment (nicholasdower) [inline markdown/dev/reference/api/config/options/en.md:30]: I believe you will also need to update line 132 like:
```diff
-[count]: /reference/api/config/options/count
+[counter]: /reference/api/config/options/counter
```

Comment (bobgeorgethe3rd) [reply markdown/dev/reference/api/config/options/en.md:30]: Done Lol of course I missed the actual link gg me

**Comment by bobgeorgethe3rd** [comment]
Noticed a few more broken links so slipped them in.

File: markdown/dev/reference/api/config/options/en.md (hunk 1 above)
Comment (joostdecock) [reply markdown/dev/reference/api/config/options/en.md:30]: What's there is not going to result in a broken link.

This is a markdown link using a reference. So:

```md
[link text][link reference]
```

Then you can define the reference elsewhere on the page:

[link reference]: /some/link/here

What's currently in this PR is this:

```md
[**counter** options][counter] are for integer values

[count]: /reference/api/config/options/counter
```

So the link reference that's defined is `count` but the reference used is `counter`.

Long story short: it seems there's some confusion between the URL and the link reference (which is just a unique ID and can be anything)

**Review by joostdecock — CHANGES_REQUESTED** [review]
These are good fixes, but the `count` link reference should be used, not `counter` (but adapting the URL is good)

**Comment by bobgeorgethe3rd** [comment]
I see what you saying have used [count] and just changed the url.

**Review by nicholasdower — APPROVED** [review]
Looks great. :shipit:

**Comment by nicholasdower** [comment]
@joostdecock it says you requested a change but I don't think that is true:
image

Going to merge but please let me know if something needs to be followed up on.

**Comment by joostdecock** [comment]
@nicholasdower 

> @joostdecock it says you requested a change but I don't think that is true:

I did, but @bobgeorgethe3rd already made the change:

> I see what you saying have used [count] and just changed the url.

All's good :+1:

**Pull request merged**